project/config/settings/production.py

```diff
@@ -63,53 +41,6 @@
     "DJANGO_SECURE_CONTENT_TYPE_NOSNIFF", default=True
 )
 
-# STORAGES
-# ------------------------------------------------------------------------------
-# https://django-storages.readthedocs.io/en/latest/#installation
-INSTALLED_APPS += ["storages"]  # noqa F405
```

Comment:
++ Removing cruft from production file! yay

Reply:
I was unsure about these as they clearly serve a purpose. I removed what I had to in order to deploy with the production configuration. It is entirely possible these will be used later. My working assumption is that we can add these back if needed.